packages/punchcard/lib/appsync/vtl.ts

```diff
@@ -0,0 +1,78 @@
+import { number as numberShape, ShapeGuards, string as stringShape } from '@punchcard/shape';
+import { Shape } from '@punchcard/shape/lib/shape';
+import { VExpression } from './expression';
+import { set, Statement } from './statement';
+import { VFloat, Visitor, VObject, VString } from './vtl-object';
+
+/**
+ * Represents a Velocity Template program.
+ */
+export type VTL<T> = Generator<Statement<VObject | void>, T>;
+
+declare module './vtl-object' {
```

Comment:
Why do you take this module augmentation approach? Can these `VObject` methods be defined within the `vtl-object` module instead?

Reply:
Was getting circular dependencies. Been meaning to come back and fix this, it just hasn't been a priority. Will fix.